reference/promotion_settings.v3.yml

```diff
@@ -0,0 +1,299 @@
+info:
+  title: Promotion Settings API
+  version: "1.0"
+  description: |
+    A *promotion setting* determines how promotions should be applied and/or calculated.
+
+    To learn more about promotions, consult the [Promotions Overview](/docs/store-operations/promotions).
+  termsOfService: "https://www.bigcommerce.com/terms"
+  contact:
+    name: BigCommerce
+    url: "https://www.bigcommerce.com"
+    email: support@bigcommerce.com
+servers:
+  - url: "https://api.bigcommerce.com/stores/{store_hash}/v3"
+    variables:
+      store_hash:
+        default: store_hash
+        description: Permanent ID of the BigCommerce store.
+    description: BigCommerce API Gateway
+security:
+  - X-Auth-Token: []
+tags:
+  - name: Promotion Settings
+paths:
+  "/promotions/settings":
+    parameters:
+      - $ref: "#/components/parameters/Accept"
+    get:
+      summary: Get Global Promotion Settings
+      description: Returns global promotion settings.
+      operationId: getGlobalPromotionSettings
+      tags:
+        - Promotion Settings
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/GlobalPromotionSettings"
+              examples:
+                EnableZeroPriceProductsToTriggerPromotions:
+                  $ref: "#/components/examples/EnableZeroPriceProductsToTriggerPromotions"
+        "400":
+          description: Bad request. Authentication Required.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse400"
+    put:
+      summary: Update Global Promotion Settings
+      description: Updates global promotion settings.
+      operationId: updateGlobalPromotionSettings
+      parameters:
+        - $ref: "#/components/parameters/ContentType"
+      tags:
+        - Promotion Settings
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/GlobalPromotionSettings"
+            examples:
+              EnableZeroPriceProductsToTriggerPromotions:
+                $ref: "#/components/examples/EnableZeroPriceProductsToTriggerPromotions"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: "#/components/schemas/GlobalPromotionSettings"
+        "400":
+          description: Bad request. Authentication Required.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse400"
+        "403":
+          description: Forbidden. Operation cannot be performed with existing privileges.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse403"
+        "422":
```

Comment:
Note for Docs team: Right now for Orders / Promotions documentation (as seen in screenshot) we aren't providing relevant error examples. What's the recommended work around for that?
<img width="1511" alt="image" src="https://github.com/user-attachments/assets/5249afec-662c-440a-9f06-742fde37ef66">

Reply:
Hi @jpajar we can add them. Is this your request?

Reply:
Hi @bc-traciporter that would be much appreciated, thanks!

Reply:
Thanks @bc-traciporter , I will add comments and more information on that ticket. In the mean time if you have any further feedback re: this open PR please lmk!

Reply:
Hey @bc-traciporter just wanted to ask if this PR needs an approval from the dev-docs team?

Reply:
Yes I will approve the PR.